Upstream-Status: Backport

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From b5984fc0f01959dbbb265495afb9b3253a42c730 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 19 Feb 2019 12:35:00 +0000
Subject: [PATCH 20/42] [ozone/wayland] Fix includes and deps.

Fix BUILD.gn includes and headers usages.

Bug: 578890
Change-Id: Ib8a7626f74d58f7584c8c44aff7f69a512c39a6d
Reviewed-on: https://chromium-review.googlesource.com/c/1477126
Reviewed-by: Antonio Gomes <tonikitoo@igalia.com>
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Cr-Commit-Position: refs/heads/master@{#633234}
---
 ui/ozone/platform/wayland/BUILD.gn                  | 7 +++----
 ui/ozone/platform/wayland/ozone_platform_wayland.cc | 2 +-
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/ui/ozone/platform/wayland/BUILD.gn b/ui/ozone/platform/wayland/BUILD.gn
index 64cf295db3b4..250250f64192 100644
--- a/ui/ozone/platform/wayland/BUILD.gn
+++ b/ui/ozone/platform/wayland/BUILD.gn
@@ -18,6 +18,8 @@ source_set("wayland") {
     "client_native_pixmap_factory_wayland.h",
     "gl_surface_wayland.cc",
     "gl_surface_wayland.h",
+    "gpu/drm_render_node_path_finder.cc",
+    "gpu/drm_render_node_path_finder.h",
     "gpu/wayland_connection_proxy.cc",
     "gpu/wayland_connection_proxy.h",
     "ozone_platform_wayland.cc",
@@ -90,7 +92,6 @@ source_set("wayland") {
     "//build/config/linux/libdrm",
     "//mojo/public/cpp/bindings",
     "//skia",
-    "//third_party/minigbm",
     "//third_party/wayland:wayland_client",
     "//third_party/wayland-protocols:linux_dmabuf_protocol",
     "//third_party/wayland-protocols:presentation_time_protocol",
@@ -125,8 +126,6 @@ source_set("wayland") {
     sources += [
       "gpu/drm_render_node_handle.cc",
       "gpu/drm_render_node_handle.h",
-      "gpu/drm_render_node_path_finder.cc",
-      "gpu/drm_render_node_path_finder.h",
       "gpu/gbm_pixmap_wayland.cc",
       "gpu/gbm_pixmap_wayland.h",
       "gpu/gbm_surfaceless_wayland.cc",
@@ -134,7 +133,7 @@ source_set("wayland") {
     ]
 
     deps += [
-      "//build/config/linux/libdrm",
+      "//third_party/minigbm",
       "//ui/gfx:memory_buffer",
       "//ui/ozone/common/linux:gbm",
     ]
diff --git a/ui/ozone/platform/wayland/ozone_platform_wayland.cc b/ui/ozone/platform/wayland/ozone_platform_wayland.cc
index a4296fefeb72..476c2a2fbd91 100644
--- a/ui/ozone/platform/wayland/ozone_platform_wayland.cc
+++ b/ui/ozone/platform/wayland/ozone_platform_wayland.cc
@@ -17,6 +17,7 @@
 #include "ui/events/system_input_injector.h"
 #include "ui/gfx/linux/client_native_pixmap_dmabuf.h"
 #include "ui/ozone/common/stub_overlay_manager.h"
+#include "ui/ozone/platform/wayland/gpu/drm_render_node_path_finder.h"
 #include "ui/ozone/platform/wayland/gpu/wayland_connection_proxy.h"
 #include "ui/ozone/platform/wayland/wayland_connection.h"
 #include "ui/ozone/platform/wayland/wayland_connection_connector.h"
@@ -40,7 +41,6 @@
 #include "ui/base/ui_base_features.h"
 #include "ui/ozone/common/linux/gbm_wrapper.h"
 #include "ui/ozone/platform/wayland/gpu/drm_render_node_handle.h"
-#include "ui/ozone/platform/wayland/gpu/drm_render_node_path_finder.h"
 #endif
 
 namespace ui {
-- 
2.17.1

